optaplanner-benchmark/pom.xml

```diff
@@ -106,6 +106,11 @@
       <groupId>org.freemarker</groupId>
       <artifactId>freemarker</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
@rsynek We need to discuss this in light of JUnit 5.

Reply:
Discussed, the first step of the JUnit 5 work (vintage on kie-parent) will be prioritized for end of November.
Meanwhile, practically speaking, we won't block assertj so this dependency addition is ok.